api/v1_projects.inc

```diff
@@ -848,6 +865,9 @@ function api_v1_project_page(string $method, array $data, array $query_params)
                 return $proof_project_page->save_and_revert(receive_project_text_from_request_body('text'));
             case 'abandon':
                 return $proof_project_page->returnToRound($pguser);
+            case 'report':
```

Comment:
I don't think this is a good fit for this endpoint. All other actions operate on the page text, perform an action on the page text, and return a page text. Adding a new endpoint specifically for the page's wordcheck would be cleaner and not require the schema juggling, eg:
```
PUT v1/projects/:projectid/pages/:pagename/wordcheck
```

Reply:
Seems reasonable.